conf/application.conf

```diff
@@ -67,7 +71,7 @@ mongodb {
 
 urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:9000/digital-service-tax-registration-frontend"
+  loginContinue = "http://localhost:8744/digital-services-tax-registration"
```

Comment:
I think the url was changed back to "service" instead of "services"

Reply:
this change should not be showing up here and i think its due to my rebase. 
Main branch :   loginContinue = "http://localhost:8744/digital-services-tax-registration